WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -132,6 +132,11 @@ public async Task<Wallet> StartWalletAsync(Wallet wallet)
 
 			lock (Lock)
 			{
+				if (CancelAllInitialization.IsCancellationRequested || CancelAllInitialization.Token.IsCancellationRequested)
```

Comment:
```suggestion
				if (CancelAllInitialization.IsCancellationRequested)
```

Those values cannot be different given implementation: https://source.dot.net/#System.Private.CoreLib/CancellationTokenSource.cs,9e22b9c8995bf195

Reply:
Good to know! Thanks.